src/github/logic.py

```diff
@@ -193,18 +211,14 @@ def pull_request_approved_after_merging(pull_request: PullRequest) -> bool:
     return False
 
 
-def all_pull_request_participants(pull_request: PullRequest) -> List[str]:
+def pull_request_participants(pull_request: PullRequest) -> List[str]:
     return list(
         set(
             gh_handle
             for gh_handle in (
                 [pull_request.author_handle()]
                 + pull_request.assignees()
-                + pull_request.reviewers()
                 + pull_request.requested_reviewers()
-                + _pull_request_commenters(pull_request)
-                + _pull_request_comment_mentions(pull_request)
-                + _pull_request_review_mentions(pull_request)
```

Comment:
from what I can tell, these helper functions aren't used anymore so I think it would be best to delete them in this PR